lib/widgets/icons.dart

```diff
@@ -33,65 +33,77 @@ abstract final class ZulipIcons {
   /// The Zulip custom icon "at_sign".
   static const IconData at_sign = IconData(0xf103, fontFamily: "Zulip Icons");
 
+  /// The Zulip custom icon "attach_file".
+  static const IconData attach_file = IconData(0xf104, fontFamily: "Zulip Icons");
+
   /// The Zulip custom icon "bot".
-  static const IconData bot = IconData(0xf104, fontFamily: "Zulip Icons");
+  static const IconData bot = IconData(0xf105, fontFamily: "Zulip Icons");
+
+  /// The Zulip custom icon "camera".
+  static const IconData camera = IconData(0xf106, fontFamily: "Zulip Icons");
 
   /// The Zulip custom icon "chevron_right".
-  static const IconData chevron_right = IconData(0xf105, fontFamily: "Zulip Icons");
+  static const IconData chevron_right = IconData(0xf107, fontFamily: "Zulip Icons");
 
   /// The Zulip custom icon "clock".
-  static const IconData clock = IconData(0xf106, fontFamily: "Zulip Icons");
+  static const IconData clock = IconData(0xf108, fontFamily: "Zulip Icons");
 
   /// The Zulip custom icon "copy".
-  static const IconData copy = IconData(0xf107, fontFamily: "Zulip Icons");
+  static const IconData copy = IconData(0xf109, fontFamily: "Zulip Icons");
 
   /// The Zulip custom icon "format_quote".
-  static const IconData format_quote = IconData(0xf108, fontFamily: "Zulip Icons");
+  static const IconData format_quote = IconData(0xf10a, fontFamily: "Zulip Icons");
 
   /// The Zulip custom icon "globe".
-  static const IconData globe = IconData(0xf109, fontFamily: "Zulip Icons");
+  static const IconData globe = IconData(0xf10b, fontFamily: "Zulip Icons");
 
   /// The Zulip custom icon "group_dm".
-  static const IconData group_dm = IconData(0xf10a, fontFamily: "Zulip Icons");
+  static const IconData group_dm = IconData(0xf10c, fontFamily: "Zulip Icons");
 
   /// The Zulip custom icon "hash_sign".
-  static const IconData hash_sign = IconData(0xf10b, fontFamily: "Zulip Icons");
+  static const IconData hash_sign = IconData(0xf10d, fontFamily: "Zulip Icons");
+
+  /// The Zulip custom icon "image".
+  static const IconData image = IconData(0xf10e, fontFamily: "Zulip Icons");
```

Comment:
Because the icon wasn't rendering correctly, I resorted to the original source. However, processing the exported icon with the [svg fixer](https://www.npmjs.com/package/oslllo-svg-fixer) seems to fix this, as I just tested.